src/parse/mod.rs

```diff
@@ -37,10 +37,15 @@ pub fn parse_program_sequence<'a,'b> (
       break;
     }
     if let Some(val) = iter.peek() { match *val {
+      // Anything written after an # is a comment and should be ignored
+      "#" => { while iter.next().unwrap_or("\n") != "\n" {} }
       // Whitespace generally has no significance, but sometimes the sub-parsers
       // may use it to identify the end of their input
       // (We need to take next to not invoke infinitely)
-      " " => { iter.next(); },
+      " " | "\n" | "\t" => { iter.next(); },
+
+      // Is this right? perhaps it's just the fizzbuzz example using it
```

Comment:
^

Reply:
It is indeed a needed bug-fix. I forgot to include newlines and tabs in the unit tests, and thus didn't catch the mistake.

Reply:
Just realized you refer to the `,` here, but you can see below what I think of it.